.github/workflows/update-release-branch.yml

```diff
@@ -14,14 +14,30 @@ jobs:
           token: ${{ secrets.GH_TOKEN }} # use PAT with permissions to push to master
           ref: release
 
-      - name: Reset release branch
+      - name: Perform reset
         run: |
           git reset --hard ${{ github.ref }}
           git push --force origin release
 
+  production-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0 # fetch history for all branches and tags
+          token: ${{ secrets.GH_TOKEN }} # use PAT with permissions to push to master
```

Comment:
Why is this token needed here? I think we should stick to the default `GITHUB_TOKEN` with limited permissions when possible

Reply:
Good catch, I agree. If using the default do I need to specify a token here? Or can the entire `token` field be left out, and it will default to `GITHUB_TOKEN`?

Reply:
Yeah, `actions/checkout` defaults to `GITHUB_TOKEN` if `token` isn't provided.